Makefile

```diff
@@ -90,30 +97,31 @@ precommit: cover gofmt lint test
 $(COMPILER_BIN) $(RUNNER_BIN) $(TOOL_BINS): build/bin/%: tools/%
 	@mkdir -p build/bin
 	@cp -f $< $@
+	@sed -i '' '1s@/usr/bin/env python@$(PYTHON_BIN)@' $@
```

Comment:
I think 'sed -i' for in place edit is not supported on many systems. I'll come up with something different.